mtgjson5/classes/mtgjson_card.py

```diff
@@ -57,6 +57,7 @@ class MtgjsonCardObject:
     has_non_foil: Optional[bool]  # Deprecated - Remove in 5.3.0
     identifiers: MtgjsonIdentifiersObject
     is_alternative: Optional[bool]
+    is_foil: Optional[bool]
```

Comment:
what about `is_etched`?

Reply:
This variable is only used for the in-line decks, and it's a legacy field for that at this time. We can look to expand upon it in the future, sure, but I'd like to keep some sense of consistency

Reply:
if you want consistency, drop `isFoil`, use `finish` and mark it as a string